src/3_api.js

```diff
@@ -156,7 +156,7 @@ Server.prototype.getUrl = function(resource, data) {
 	}
 	return {
 		data: this.serializeObject(d, ''),
-		url: url
+		url: url.replace(/^\//, '')
```

Comment:
Because `getUrl` joins url fragments with `/`, and because we no longer have a destination for the `linkClick` resource, we need to get rid of the extra slash that can be prepended.